src/Apps/Artwork/Components/ArtworkAuctionCreateAlertHeader/ArtworkAuctionCreateAlertHeader.tsx

```diff
@@ -0,0 +1,74 @@
+import { Box, Text } from "@artsy/palette"
+import * as React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { ArtworkAuctionCreateAlertHeader_artwork$data } from "__generated__/ArtworkAuctionCreateAlertHeader_artwork.graphql"
+import { ArtworkSidebarCreateAlertButtonFragmentContainer } from "Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCreateAlertButton"
+import { useTimer } from "Utils/Hooks/useTimer"
+import { useFeatureFlag } from "System/useFeatureFlag"
+import { lotIsClosed } from "Apps/Artwork/Utils/lotIsClosed"
+
+interface ArtworkAuctionCreateAlertHeaderProps {
+  artwork: ArtworkAuctionCreateAlertHeader_artwork$data
+}
+
+const ArtworkAuctionCreateAlertHeader: React.FC<ArtworkAuctionCreateAlertHeaderProps> = ({
+  artwork,
+}) => {
+  const hasArtists = (artwork.artists?.length ?? 0) > 0
+  const biddingEndAt =
+    artwork?.saleArtwork?.extendedBiddingEndAt ?? artwork?.saleArtwork?.endAt
+  const { hasEnded } = useTimer(biddingEndAt!, artwork?.sale?.startAt!)
+
+  const auctionHeaderAlertCTAEnabled = useFeatureFlag(
+    "onyx_auction-header-alert-cta"
+  )
+
+  const isLotClosed = hasEnded || lotIsClosed(artwork.sale, artwork.saleArtwork)
+  const displayAuctionCreateAlertHeader =
+    hasArtists &&
+    artwork.isInAuction &&
+    isLotClosed &&
+    auctionHeaderAlertCTAEnabled
+
+  if (!displayAuctionCreateAlertHeader) return null
+
+  const artistName = artwork.artistNames ? ", " + artwork.artistNames : ""
+
+  return (
+    <Box py={12}>
+      <Text variant="lg" textAlign="center">
+        Bidding for <i>{artwork.title?.trim()}</i>
+        {artistName} has ended.
+      </Text>
+      <Box mt={2} mx="auto" width={["100%", 209]}>
```

Comment:
What's with the 209?

Reply:
I'm not sure! I think this was added based on something from the figma spec? cc @tam-kis (either way this can be tweaked later on)

Reply:
Oh this is a button that should align with the grid. This is a classic example of misreading the Figma, though I'm pretty sure there's no way to turn on the grid. What you would really do here is something like:

```
<GridColumns>
  <Column span={12}>
      <Text variant="lg" textAlign="center">
        Bidding for <i>{artwork.title?.trim()}</i>
        {artistName} has ended.
      </Text>
  </Column>

  <Column span={2} start={5}>
    <ArtworkSidebarCreateAlertButtonFragmentContainer artwork={artwork} />
  <Column>
</GridColumns>

Reply:
Yes exactly, I was unsure about that but also saw some places where we do add a width. I am waiting for some design clarifications for the button and will update it async.

Reply:
👍 

Yeah, in general we can just assume we want alignment to the grid. Easy rule to avoid magic numbers for widths and heights.